nautobot/ipam/querysets.py

```diff
@@ -10,39 +10,53 @@
 
 
 class NetworkQuerySet(QuerySet):
+    @staticmethod
+    def _split_prefix(prefix):
+        if isinstance(prefix, str):
+            prefix = netaddr.IPNetwork(prefix)
+        broadcast = prefix.broadcast
+        if prefix.broadcast is None:
+            broadcast = prefix.network
+        return prefix.prefixlen, prefix.network, broadcast
```

Comment:
Since this is returning a tuple, can you please make it return a `collections.namedtuple` instead?

See: https://gist.github.com/lucasnad27/e93b8f316259be3299e5#clarify-multiple-return-values-with-named-tuples